Makefile

```diff
@@ -51,7 +51,7 @@ test: envtest ## Run tests.
 
 .PHONY: add-license
 add-license: addlicense ## Add license headers to all go files.
-	find . -name '*.go' -exec $(ADDLICENSE) -c 'OnMetal authors' {} +
+	find . -name '*.go' -exec go run github.com/google/addlicense -f hack/license-header.txt {} +
```

Comment:
What is the reason to change `-exec $(ADDLICENSE)` to `-exec go run github.com/google/addlicense`? Taking into account that we will download it anyways: `add-license: addlicense`

Reply:
Didn't see that `addlicense` target. Consider it c&p error, fixed.